JASP-Engine/JASP/R/anovarepeatedmeasures.R

```diff
@@ -1252,6 +1245,12 @@ AnovaRepeatedMeasures <- function(jaspResults, dataset = NULL, options) {
         contrastResult[["df"]]      <- sapply(allTestResults, function(x) x[["parameter"]])
         contrastResult[["SE"]]      <- sapply(allTestResults, function(x) x[["estimate"]] /  x[["statistic"]])
         contrastResult[["p.value"]] <- sapply(allTestResults, function(x) x[["p.value"]])
+        
+      } else if (options$contrastAssumeEqualVariance == FALSE) {
+        
+        contrastContainer[[paste0(contrast$contrast, "Contrast_",  contrast$variable)]]$setError(gettextf("Unequal variances only available for within subjects factors"))
```

Comment:
```suggestion
        contrastContainer[[paste0(contrast$contrast, "Contrast_",  contrast$variable)]]$setError(gettext("Unequal variances only available for within subjects factors"))
```
`gettext` rather than `gettextf`.